spk/minio/src/wizard/install_uifile

```diff
@@ -0,0 +1,64 @@
+[
+	{
+		"step_title": "MinIO configuration",
+		"items": [
+		  {
+			"type": "textfield",
+			"subitems": [
+			  {
+				"key": "wizard_data_directory",
+				"desc": "Data directory location",
+				"defaultValue": "/volume1/minio",
+				"validator": {
+				  "allowBlank": false,
+				  "regex": {
+					"expr": "/^\\\/volume[0-9]{1,2}\\\/[^<>: */?\"]*/",
+					"errorText": "Path should begin with /volume?/ where ? is volume number (1-99)"
+				  }
+				}
+			  },
+			  {
+				"key": "wizard_access_key",
+				"desc": "MinIO access key",
+				"defaultValue": "minio-access-822791549661344",
+				"validator": {
+				  "allowBlank": false,
+				  "minLength": 3,
+				  "regex": {
+					"expr": "/^[^<>:*/?\"]*$/",
```

Comment:
@fgma you might want to exclude `|` since it will break your `sed` in `service-setup`, no? Same for `Secret Key`.

Reply:
Fixed in https://github.com/SynoCommunity/spksrc/pull/3689/commits/237c60d8d6c80c7a7498c81fd9a1716c360e3d05